lib/rubocop/cop/style/implicit_array.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for arrays that are not explicitly initialized with
+      # brackets. Exceptions are allowed for implicit arrays of exceptions in a
+      # rescue block and for the result of a splat operator.
+      #
+      # @example
+      #   # bad
+      #   a = 1, 2, 3
+      #
+      #   # good
+      #   a = [1, 2, 3]
+      #
+      # @example
+      #
+      #   # good
+      #   def foo
+      #     some_scary_method
+      #   rescue SomeException, SomeOtherException
+      #     some_cleanup
+      #   end
+      #
+      # @example
+      #
+      #   # good
+      #   x = *y
+      class ImplicitArray < Cop
+        MSG = 'Explicitly initialize arrays with `[` and `]`.'.freeze
+
+        def on_array(node)
+          return if node.parent.resbody_type?
```

Comment:
I guess the cop crashes on an array in the top. Because an array in the top does not have `parent`. For example:

```ruby
[1, 2, 3]
```

Can you add a test case for this and fix the problem if it crashes?

Reply:
Nice catch. Done.